binding.gyp

```diff
@@ -28,13 +29,16 @@
                 ],
               }],
               ["OS=='mac'", {
-                "cflags+": ["-fvisibility=hidden"],
-                "xcode_settings": {
-                  "GCC_SYMBOLS_PRIVATE_EXTERN": "YES",  # -fvisibility=hidden
-                  "GCC_ENABLE_CPP_EXCEPTIONS": "YES",
-                },
-                "libraries": [ "-lzim" ]
-              }],
+                    "include_dirs": [
+                        "<(libzim_include)",
+                    ],
+                    "cflags+": ["-fvisibility=hidden"],
+                    "xcode_settings": {
```

Comment:
I don’t really understand why this is so different from Linux. Considering, this is the same compiler.